SPF5122Z RF ampllifier

chmorgan opened the pull request:
*Replace this line with your commit message! Please provide a description of your pull request*

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] Provide a screenshot of the symbol(s) from the symbol editor with the pin types visible
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

---

Be patient, we maintainers are volunteers with limited time and need to check your contribution against the datasheet. You can speed up the process by providing all the necessary information (see above). And you can speed up the process even more by providing additional info like the screenshot of the symbol editor pin table (or for high pin counts converted to csv) sorted in the same way as the pin table in the datasheet and a direct link to the datasheet page that contains the pin table.

myfreescalewebpage commented:
Hi @chmorgan , thanks for contributing,

A few comments I have during my review:

- [ ] Description should ends with package name ", DFN-8"
- [ ] keywords in lower case only is great
- [ ] Footprint filter should be `DFN*1EP*2x2mm*P0.5mm*`
- [ ] Pin 3 and 6 should be stacked with GND

Cheers,
Joel

chmorgan commented:
Hi @myfreescalewebpage. Thanks for yet another review!

I fixed the footprint filter, keywords case and added the package to the description. As this is a high frequency RF part I've been assuming that the mfg footprint recommendations were the best way to achieve performance. Even though 3 and 6 are NC pins the mfg recommends tying them together with the adjacent pins as I've done.

See the two below images cut from the datasheet:

![image](https://user-images.githubusercontent.com/1211904/88106856-2a077c80-cb74-11ea-81b9-b0ce79e53cc8.png)

![image](https://user-images.githubusercontent.com/1211904/88106882-31c72100-cb74-11ea-8646-39429adbd01f.png)

myfreescalewebpage commented:
Hi @chmorgan with pleasure :)
Thanks for pointing this details, accepted.
Just need to wait the footprint.
Joel
